frontend/service/agents/PredictTrader.ts

```diff
@@ -16,6 +16,17 @@ import { ONE_YEAR, StakedAgentService } from './StakedAgentService';
 
 const MECH_REQUESTS_SAFETY_MARGIN = 1;
 
+// TODO: move to common place, enums/StakingProgram.ts?
+const PREDICT_TRADER_STAKING_PROGRAM_IDS = [
```

Comment:
We have the list of staking programs, it's just based on chain, not agent type. Maybe should rethink this approach, but don't think we should duplicate